tests/tests_unit.yml

```diff
@@ -29,18 +30,31 @@
       ignore_errors: true
       register: python2_available
 
-    - command: python2 /tmp/test-unit-1/test_network_connections.py
+    - command: python2 /tmp/test-unit-1/test_network_connections.py --verbose
       when: python2_available | succeeded
-
+      register: python2_result
```

Comment:
if this command fails, I presume it ignores the error and proceeds with the other tests, right? That makes sense.

but at the end of the play, after all the debug messages, don't you lack a final command to fail the entire play/test, if any of the `python*_results` is not success?

Reply:
It will run when python2 is available and fail when the tests fail. Then the ```python_results``` are not used. I added them to be able to see if/which tests are skipped.